.gitignore

```diff
@@ -3,3 +3,8 @@ target/
 build.bat
 .project
 .classpath
+update eclipse.bat
+src/main/java/net/laboulangerie/laboulangeriecore/commands/CountdownCommand.java
```

Comment:
What is this?

Reply:
c'est qu'une fois ze m'avait demandé de voir pour un countdown pour l'ouverture, mais finalement ça a été laisser tombr et j'ia oublié de modifier le gitignore ^^"